### PR TITLE
Remove custom Ajax file loader

### PR DESCRIPTION
**Pull request by bbecquet**
Now that the map bundle async loading is done through webpack, the custom file loader is used only to load the language file, and we don't need two implementations. Let's simplify!

### CODE REVIEW

File: dev/performances.js [hunk 1]
```diff
@@ -106,8 +106,7 @@ async function serverStart() {
 
   configBuilder.set('mapStyle:poiMapUrl', `["http://localhost:${PORT}/fake_pbf/{z}/{x}/{y}.pbf"]`)
   configBuilder.set('mapStyle:baseMapUrl', `["http://localhost:${PORT}/fake_pbf/{z}/{x}/{y}.pbf"]`)
-  configBuilder.set('system:evalFiles', false)
-  configBuilder.set('performance:enabled', true)
+    configBuilder.set('performance:enabled', true)
```
Comment (amatissart) [inline dev/performances.js]: extra spaces

File: src/libs/async_file_loader.js [hunk 2]
```diff
@@ -1,14 +1,10 @@
...
-    fileLoader.onLoad = () => {
-      resolve();
-    };
-    if (options.onProgress) {
-      fileLoader.onProgress = this.onProgress;
-    }
+    const sc = document.createElement('script');
+    sc.src = uri;
+    document.body.appendChild(sc);
+    sc.onload = resolve;
```
Comment (amatissart) [inline src/libs/async_file_loader.js]: Wouldn't it be safer to set `onload` before calling `appendChild` ?

File: dev/performances.js (hunk 1 above)
Comment (bbecquet) [reply dev/performances.js]: Thanks. Fixed!

File: src/libs/async_file_loader.js (hunk 2 above)
Comment (bbecquet) [reply src/libs/async_file_loader.js]: Indeed. I changed it.

**Pull request merged**